man/subscription-manager.8

```diff
@@ -276,7 +276,7 @@ This command has no options.
 .SS ATTACH OPTIONS
 The
 .B attach
-command applies a specific subscription to the system.
+command applies a specific subscription to the system. This command is not possible to use, when the content access mode is set to simple content access mode.
```

Comment:
I'd suggest:
"... when the content access mode of the organization to which the system is registered is simple content access mode"

Is there a kbase we could reference to provide additional detail on SCA?